products/libllbuild/BuildDB-C-API.cpp

```diff
@@ -166,7 +166,7 @@ class CAPIBuildDB: public BuildDBDelegate {
 
 const llb_data_t mapData(std::vector<uint8_t> input) {
   const auto data = input.data();
-  const auto size = sizeof(data) * input.size();
+  const auto size = sizeof(input.front()) * input.size();
   const auto newData = (uint8_t *)malloc(size);
   memcpy(newData, data, size);
```

Comment:
Can you also change this use of `data` to just be `input.data()` and drop the local too, since I think now it is unused?

Reply:
Done 👍